### ISSUE TITLE
pre_train" and "alt_train" 

### ISSUE THREAD

**Issue by 11051911** [WHY-17/scDSSC#1]
Hello, I want to ask what is the difference between "pre_train" and "alt_train" in the code, I see that the code of the two is exactly the same, is there any difference in the function?
                                                                                                                                                                                   Yours sincerely.

**Comment by WHY-17** [comment]
learning rate.



---- Replied Message ----
| From | ***@***.***> |
| Date | 08/29/2023 10:49 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | [WHY-17/scDSSC] pre_train" and "alt_train" (Issue #1) |

Hello, I want to ask what is the difference between "pre_train" and "alt_train" in the code, I see that the code of the two is exactly the same, is there any difference in the function?
Yours sincerely.

**Comment by 11051911** [comment]
> learning rate.
> […](#)
> ---- Replied Message ---- | From | ***@***.***> | | Date | 08/29/2023 10:49 | | To | ***@***.***> | | Cc | ***@***.***> | | Subject | [WHY-17/scDSSC] pre_train" and "alt_train" (Issue #1) | Hello, I want to ask what is the difference between "pre_train" and "alt_train" in the code, I see that the code of the two is exactly the same, is there any difference in the function? Yours sincerely. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
This is a very good paper. Why do you want to set it this way ("pre_train" and "alt_train"), I found that the Coef_1 I got in the process of debugging the program is the same as the Coef_2, or what are the advantages of setting "pre_train" and "alt_train" (only one of them has any defects)? This place seems to bother me.

**Comment by WHY-17** [comment]
I don't know where the bug appears. Maybe it's a conflict between some installation packages, because it used to work.



---- Replied Message ----
| From | ***@***.***> |
| Date | 08/29/2023 11:03 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [WHY-17/scDSSC] pre_train" and "alt_train" (Issue #1) |

learning rate.
…
---- Replied Message ---- | From | @.> | | Date | 08/29/2023 10:49 | | To | @.> | | Cc | @.> | | Subject | [WHY-17/scDSSC] pre_train" and "alt_train" (Issue #1) | Hello, I want to ask what is the difference between "pre_train" and "alt_train" in the code, I see that the code of the two is exactly the same, is there any difference in the function? Yours sincerely. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: @.>
This is a very good paper. Why do you want to set it this way ("pre_train" and "alt_train"), I found that the Coef_1 I got in the process of debugging the program is the same as the Coef_2, or what are the advantages of setting "pre_train" and "alt_train" (only one of them has any defects)? This place seems to bother me.

**Comment by WHY-17** [comment]
In pretraining stage, I set a bigger learning rate and try to approach the optimal solution quickly. In next stage, a smaller learning rate could help us find a more accurate solution.
I hope this was helpful and thank you for reading our paper.
Best wishes!





---- Replied Message ----
| From | ***@***.***> |
| Date | 08/29/2023 11:03 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [WHY-17/scDSSC] pre_train" and "alt_train" (Issue #1) |

learning rate.
…
---- Replied Message ---- | From | @.> | | Date | 08/29/2023 10:49 | | To | @.> | | Cc | @.> | | Subject | [WHY-17/scDSSC] pre_train" and "alt_train" (Issue #1) | Hello, I want to ask what is the difference between "pre_train" and "alt_train" in the code, I see that the code of the two is exactly the same, is there any difference in the function? Yours sincerely. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: @.>
This is a very good paper. Why do you want to set it this way ("pre_train" and "alt_train"), I found that the Coef_1 I got in the process of debugging the program is the same as the Coef_2, or what are the advantages of setting "pre_train" and "alt_train" (only one of them has any defects)? This place seems to bother me.
